### ISSUE TITLE
Unable to install on Centos9

### ISSUE THREAD

**Issue by JediNight** [pjmp/bropages#7]
Trying to install on Centos Stream 9. When i run `cargo install bropages` 

```
Caused by:
  process didn't exit successfully: `/tmp/cargo-installS7Vh2z/release/build/bropages-5db906cb1f09dfa0/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', /home/student/.cargo/registry/src/github.com-1ecc6299db9ec823/bropages-1.40.0/build.rs:7:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `bropages v1.40.0`, intermediate artifacts can be found at `/tmp/cargo-installS7Vh2z`
```

When i run `cargo install --git https://github.com/pombadev/bropages.git`, i get virtually the same error
```
Caused by:
  process didn't exit successfully: `/tmp/cargo-install2G52JH/release/build/bropages-8182132bf134a16d/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', build.rs:7:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `bropages v1.40.0 (https://github.com/pombadev/bropages.git#aa6c2f24)`, intermediate artifacts can be found at `/tmp/cargo-install2G52JH

```

**Comment by pombadev** [comment]
The [build script runs `git`](https://github.com/pombadev/bropages/blob/master/build.rs#L4) while building `bropages`.
For now make sure you have git binary installed and is available in `PATH`.
We could change the build logic to not panic if something fails while building. PR welcomed!

**Comment by pombadev** [comment]
Closed by 1947174551d270aefb6dde54450a830aae10ee78
